components/Navbar.js

```diff
@@ -0,0 +1,72 @@
+/* eslint-disable @next/next/no-html-link-for-pages */
```

Comment:
You really need to deactivate eslint at this point?

Reply:
No, I just tried something there and forgot to delete it.